pkg/ui/workspaces/cluster-ui/src/statementDetails/planDetails/planDetails.tsx

```diff
@@ -121,7 +121,8 @@ function ExplainPlan({
   onChangeSortSetting,
 }: ExplainPlanProps): React.ReactElement {
   const explainPlan =
-    plan.explain_plan === "" ? "unavailable" : plan.explain_plan;
+    `Plan Gist: ${plan.stats.plan_gists[0]} \n\n` +
```

Comment:
Do we need to check that `.plan_gists[0]` exists or is it guaranteed to be populated?

Reply:
is guarantee, is what was used to create this page, only comes here if has the value